### ISSUE TITLE
Arc Furnace not consuming all ingredients

### ISSUE THREAD

**Issue by md5i** [DarkPacks/SevTech-Ages#3718]
## Bug Report
When processing recipes that require multiple inputs on the left-hand side of the arc furnace, such as Quartz Enriched Steel and Bronze, only one of the inputs will be consumed instead of all of them.  For example, three copper ingots combined with one tin dust results in four bronze ingots, but two copper ingots are left in the input side of the arc furnace.

## Expected Behaviour
The entire recipe is consumed.

## Possible Solution
I see the crafttweaker scripts have been changed to use a function in immersiveEngineering.zs.  I don't see anything wrong with it, but it may be worth trying without the convenience function to see if that is causing the problem, either due to a bug in crafttweaker or a misunderstanding in how the arguments are handled.

## Steps to Reproduce (for bugs)

1. Place three copper ingots in the left-hand side of an arc furnace, and one tin dust in the right.
2. Four bronze ingots will be produced, and two copper ingots are left in the furnace.

## Client Information

* Modpack Version: 3.1.1
* Java Version: 1.8.0_212
* Launcher Used: MultiMC 5
* Memory Allocated: 16GB
* Server/LAN/Single Player: Single player
* Optifine Installed: No
* Shaders Enabled: No

**Comment by md5i** [comment]
I have verified that this also takes place with the glass pane and quartz glass recipes.  These, along with the Quartz Enriched Steel and Bronze Ingot recipes are the only ones that require stacks on the left hand side.

**Comment by sk2048** [comment]
Thank you for this, it appears to be a bug on IE's side. I've submitted a report, hopefully it'll be resolved soon :)

BluSunrize/ImmersiveEngineering#3442

**Comment by md5i** [comment]
Do be careful.  In the bug report you submitted, your proposed fix mentions "when additives are present."  The aforementioned glass pane recipe that has the same bug has no additives.

**Comment by sk2048** [comment]
I assumed that because the line I suggested changing was behind an `if(consumedAdditives!=null` that the line I proposed changing would only apply where there were consumed additives. As a result, when I tested recipes without additives after applying my fix I thought I was correct in assuming that it would not affect them.

That's not the case, consumedAdditives is an empty array when no additives are consumed. Very weird. Good news is my proposed fix sorts both out.
